db/jambones-sql.sql

```diff
@@ -516,6 +516,7 @@ fallback_speech_synthesis_label VARCHAR(64),
 fallback_speech_recognizer_vendor VARCHAR(64),
 fallback_speech_recognizer_language VARCHAR(64),
 fallback_speech_recognizer_label VARCHAR(64),
+env_vars BLOB,
```

Comment:
env_vars is plain text, can we use MEDIUM TEXT here ?

Reply:
I think MEDIUM_TEXT has a size limit that could be problematic

Reply:
BLOB is for saving binary file which has limit 64KB

We store text completely which TEXT(64KB) as well.

query blob will be slower than TEXT.

Storing text to blob is possible but not good in my point of view.

Reply:
OK lets change to MEDIUMTEXT as you suggest then